packages/react/src/components/Icon/icons/plus-circle.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.2" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" width="15" height="15"><path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z" fill="currentColor" vector-effect="non-scaling-stroke"/></svg>
```

Comment:
Something to double-check with @awpearlm , but it's interesting that the api key add button is just the "+"

![image](https://user-images.githubusercontent.com/6416791/151013096-32501ed8-07e0-4144-831d-9a68882f0179.png)

Reply:
@awpearlm said the plus icon is just fine. I am closing this pull request.